src/tools/tests/schema2graph-test.ts

```diff
@@ -89,16 +106,18 @@ describe('schema2graph', () => {
         d2: reads * {a: Text, b: Text}
         d4: reads * {a: Text, b: Text, c: Text}
     `);
-    const res = convert(new SchemaGraph(manifest.particles[0]));
+    const res = convert(new SchemaGraph(manifest.particles[0], dummyNameGenerator, dummyAliasGenerator));
     assert.deepStrictEqual(res.nodes, [
-      {name: 'A_H1',       parents: '',                 children: 'AInternal1, A_H3'},
-      {name: 'AInternal1', parents: 'A_H1',             children: 'AInternal2'},
-      {name: 'A_H3',       parents: 'A_H1',             children: 'AInternal2'},
-      {name: 'AInternal2', parents: 'AInternal1, A_H3', children: ''},
+      {name: 'A_H1',       parents: '',                 children: 'A_H2, A_H3'},
+      {name: 'A_H2', parents: 'A_H1',             children: 'A_H4'},
+      {name: 'A_H3',       parents: 'A_H1',             children: 'A_H4'},
+      {name: 'A_H4', parents: 'A_H2, A_H3', children: ''},
     ]);
     assert.deepStrictEqual(res.aliases, {
-      'AInternal1': ['A_D2', 'A_H2'],
-      'AInternal2': ['A_D4', 'A_H4'],
+      'A_H1': ['A_H1'],
+      'A_H2': ['A_D2', 'A_H2'],
+      'A_H3': ['A_H3'],
+      'A_H4': ['A_D4', 'A_H4'],
```

Comment:
This seems a bit odd to me. IIUC the first node created for a given schema ends up supplying the class's actual name, and subsequent nodes are aliased to that. That suggests some sort of priority or precedence, which doesn't exist. I think having the "internal" name with all classes aliased to it is preferable.

Reply:
This is because the aliases are now generated by the Kotlin/C++ implementations. Earlier Cameron suggested I make the dummy function in this test simpler, so I made it just pass through the connections.

Reply:
I presume this will have to back to the previous state once the alias-based interim step has been removed?

Reply:
Yes.